media/css/firefox/enterprise.scss

```diff
@@ -0,0 +1,230 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import '../hubs/sections';
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+$color-photon-blue: #0099db;
+
+#outer-wrapper {
+    background: #fff;
+}
+
+.content-section {
+    color: #000;
+    padding: 40px 0;
+    text-align: center;
+
+    @media #{$mq-desktop} {
+        padding: 50px 0;
+        text-align: left;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Top hero section
+
+.intro {
+    background: #0f257e url('/media/img/firefox/enterprise/wave.png') bottom left no-repeat;
+
+    h1,
+    h2 {
+        @include font-size-level1;
+        @include open-sans;
+        font-weight: normal;
+        color: #fff;
+        margin-bottom: 0;
+    }
+
+    .copy {
+        @include open-sans;
+        color: #fff;
+        padding: 0;
+        width: auto;
+
+        p {
+            margin: 1.25em 0;
+        }
+
+        .button {
+            margin-top: 30px;
+        }
+    }
+
+    @media #{$mq-phone-wide} {
+        background: #0f257e url('/media/img/firefox/enterprise/wave.png') bottom left no-repeat;
+        background: url('/media/img/firefox/enterprise/wave.png') bottom -100px left no-repeat,
+                    url('/media/img/firefox/enterprise/circle.png') top -40px right no-repeat,
+                    linear-gradient(90deg, #003da7, #181562);
+    }
+
+    @media #{$mq-desktop} {
+        background: #0f257e url('/media/img/firefox/enterprise/wave.png') bottom left no-repeat;
+        background: url('/media/img/firefox/enterprise/laptop.png') top 150px left 600px no-repeat,
```

Comment:
Saw a comment in the bug to this effect - can we add a high res media query to load a larger version of `laptop.png`?

Reply:
I'm waiting for a higher resolution version of the image but they've asked us to push today. We can fix that after if they provide one.

Reply:
Got the image.